styles/npm/catppuccin.user.css

```diff
@@ -135,6 +135,15 @@
       color: @base;
     }
 
+    /* Syntax highlighting text */
+    span[style*="color: rgb(223, 80, 0);"] {
+      color: @peach !important;
+    }
+    
+    span[style*="color: rgb(99, 163, 92);"] {
+      color: @green !important;
+    }
```

Comment:
Why put it here? Pretty sure I already have a section somewhere for all other syntax highlighting. Be sure to follow the organization pattern of each userstyel you contribute to.

Reply:
i didnt know there was a section for syntax highlighting but i'll move it there anyway

Reply:
im not going to be like last time when i only moved the comment

Reply:
A command f search for "syntax" would find you the section. Take the time to do things properly.